### PR TITLE
feat(data_model): connects-to annotation should refer to the instance label

### PR DESCRIPTION
**Pull request by karthikjeeyar**
Changes includes:

1. The connects-to annotation should refer the instance label
2. Handles multiple edges for a node
3. yaml file changes to denote multiple edge values in connects-to annotation

### CODE REVIEW

File: frontend/public/extend/devconsole/shared/yamls/applications/analytics/deployment.yaml [hunk 1]
```diff
@@ -9,8 +9,7 @@ metadata:
     app.kubernetes.io/instance: user-analytics
     app.kubernetes.io/version: "1.0"
   annotations:
-    app.openshift.io/connects-to: 'nodejs'
-    app.openshift.io/connects-to: 'wit-deployment'
+    app.openshift.io/connects-to: '["nodejs","wit"]'
```
Comment (christianvogt) [inline frontend/public/extend/devconsole/shared/yamls/applications/analytics/deployment.yaml]: Why is there `nodejs` here? Shouldn't this be a single connection to `wit` only?

Comment (karthikjeeyar) [reply frontend/public/extend/devconsole/shared/yamls/applications/analytics/deployment.yaml]: Correct, It should be wit alone, I have created an application (nodejs) from UI and manually added the instance label (app.kubernetes.instance/nodejs) to it. I have added the label nodejs here to test whether multiple edges are getting created for the same node. now that I know it is creating multiple edges for a node, I will remove this.

File: frontend/public/extend/devconsole/shared/yamls/applications/wit/service.yaml [hunk 2]
```diff
@@ -6,14 +6,14 @@ metadata:
     app.kubernetes.io/part-of: application-1
```
Comment (sspeiche) [inline frontend/public/extend/devconsole/shared/yamls/applications/wit/service.yaml:6]: Why are the samples in the console repo when they are at https://github.com/redhat-developer/topology-example as well? 

Which one is "right"?

/cc @christianvogt @sbose78

Comment (christianvogt) [reply frontend/public/extend/devconsole/shared/yamls/applications/wit/service.yaml:6]: From an earlier conversations with @karthikjeeyar this was temporary as he was doing the work in our repository and needed immediate changes for testing. But I agree that we should only be using the samples from the example git repo.

@karthikjeeyar work with @sbose78 to ensure the git repo sample is up to date and then make sure to remove those from our repo.

**Review by christianvogt — APPROVED** [review]

File: frontend/public/extend/devconsole/shared/yamls/applications/wit/service.yaml (hunk 2 above)
Comment (karthikjeeyar) [reply frontend/public/extend/devconsole/shared/yamls/applications/wit/service.yaml:6]: These yaml files are temporary. It helps us to create and test the resources locally. I will be removing these files once the changes are up to date with the topology-example repo.

**Pull request merged**